tests/dummy/app/controllers/nodes/detail/files/provider/file.js

```diff
@@ -5,6 +5,21 @@ export default Ember.Controller.extend({
         reloadFiles() {
             this.transitionToRoute('nodes.detail.files.provider',
                 this.get('node'), this.model.get('provider'));
+        },
+        addATag(tag) {
+            var file = this.get('model');
+            var currentTags = file.get('tags').slice(0);
```

Comment:
This was the first thing I tried.  I remember it saying that pushObject was not a function, so I changed it to `Ember.A(file.get('tags')).pushObject(tag)`.  But this results in `changedAttributes` being an empty object.  So no PATCH request is sent.